src/public/js/rs.js

```diff
@@ -21,6 +105,94 @@ var RS = (function() {
     }, '');
   }
 
+  // Encodes using systematic BCH Reed-Solomon on Galois Field 256
```

Comment:
ok! I don't understand enough about "GF(256) only exists using GF(2) as its base field" but will read more to figure that out

(do the integers mod 256 use GF(2) as a base field?)